### PR TITLE
Serverless plugin

### PR DESCRIPTION
**Pull request by eoinsha**
- Remove python-based SAR app
- Replicate same functionality (Lambda functions only) in a Serverless Framework plugin
- Update build and test configuration for Node.js

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,27 +1,31 @@
...
+      - node/install-packages:
+          app-dir: ~/slic-watch/serverless-plugin
+      - run: npm run cov
       - coveralls/upload:
-          path_to_lcov: ./.coverage
-
+          flag_name: << parameters.node-version >>
 workflows:
-  main:
+  workflow:
```
Comment (guidr) [inline .circleci/config.yml:26]: Should we add a job for the npm publish here too?

**Review by guidr — COMMENTED** [review]
Few comments added

File: serverless-plugin/index.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+const DEFAULT_PLUGIN_CONFIG = {
+  alarmPeriod: 60,
+  durationPercentTimeoutThreshold: 95,
+  errorsThreshold: 0,
+  throttlesPercentThreshold: 0,
+}
+
+class ServerlessPlugin {
+  constructor(serverless, options) {
+  debugger
```
[the remark this answers is not in the document]
Comment (eoinsha) [reply serverless-plugin/index.js]: Resolved and added automatic linting with pre-commit

File: .circleci/config.yml (hunk 1 above)
Comment (eoinsha) [reply .circleci/config.yml:26]: Agreed. I'll create an issue to do this separately.

**Review by guidr — APPROVED** [review]

**Pull request merged**